pobi.md

```diff
@@ -1,41 +1,41 @@
 
 ### High Level Description
 The high level goals of the POBI protocol are:
 
-1. Each round, to distribute the sequencer function fairly among all the active registered aggregators.
-2. To synchronise the L2 round duration to L1 rounds. Because the L1 is the source of truth, the finality of the L2 transactions is dependent on the finality of the L1 rollup transaction that includes them, which means there is no advantage in publishing multiple rollups in a single L1 block. It is not possible to decrease the finality time below that of the L1. On the other hand, publishing L2 rollups less frequently means that L2 finality will be unnecessarily long. The optimum frequency is to publish one rollup per L1 block.
+1. Each round, distribute the sequencer function fairly among all the active registered aggregators.
+2. To synchronise the L2 round duration to L1 rounds. Because the L1 is the source of truth, the finality of the L2 transactions is dependent on the finality of the L1 rollup transaction that includes them, which means there is no advantage in publishing multiple rollups in a single L1 block. It is not possible to decrease the finality time below that of the L1. On the other hand, publishing L2 rollups less frequently mean that L2 finality is unnecessarily long. The optimum frequency is to publish one rollup per L1 block.
 
-To achieve fairness, the PoBI protocol states that each round the TEE can generate one random nonce and the winner of a round is the aggregator whose TEE generated the lowest random number from the group. The TEEs will generate these numbers independently and then will gossip them. The aggregators who didn't win the round, similar to L1 miners, will respect this decision because it is the rational thing to do based on the incentive mechanism. If they don't want to respect the protocol, they are free to submit a losing rollup to the L1, but it will be ignored by all compliant aggregators, which means such an aggregator has to pay L1 gas and not get any useful reward.
+To achieve fairness, the PoBI protocol states that the TEE can generate one random nonce each round, and the winner of a round is the aggregator whose TEE generates the lowest random number from the group. The TEEs generate these numbers independently and then gossip them. The aggregators who didn't win the round, similar to L1 miners, respect this decision because it is rational to do based on the incentive mechanism. If they don't respect the protocol, they are free to submit a losing rollup to the L1, which is ignored by all compliant aggregators, meaning such an aggregator has to pay L1 gas and not get any useful reward.
```

Comment:
"and then **gossips** them"

Reply:
I would say gossip is correct as its in the context of multiple TEEs?

Reply:
Agreed - ignore my comment